### PR TITLE
Added new labels to tweet_labels.py

### PR DESCRIPTION
**Pull request by pardaz-banu**
Added two new labels in the tweet_labels.py also removed the extra space present in between the word.

### CODE REVIEW

File: words_dir/tweet_labels.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
     'straight': 'Heterosexual, sexually and/or emotionally attracted to members of the opposite sex.',
     'aegosexual': 'lacking any desire to be a participant in the sexual activities therein',
     'abrosexual': 'Abrosexual refers to an individual whose sexuality is changing or fluid. For example, someone could be gay one day, then be asexual the next, then polysexual the next.',
-
+    'aromatic': 'An adjective used to describe the people who do not experience romantic attraction.',
```
Comment (VickiLanger) [inline words_dir/tweet_labels.py]: Thanks for adding this. Would you mind fixing up the spelling of `aromantic`? 

That is, unless you meant 
`'aromatic': 'One who has a pleasant or distinct smell.',` lol

**Review by VickiLanger — CHANGES_REQUESTED** [review]
Thanks for adding this. Would you mind fixing up the spelling of aromantic?

File: words_dir/tweet_labels.py (hunk 1 above)
Comment (pardaz-banu) [reply words_dir/tweet_labels.py]: Yes, i will fix up the spelling

**Review by VickiLanger — APPROVED** [review]
Thanks for contributing!

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/LGBTQotd) to see the hard work you put in.*

**Pull request merged**